### PR TITLE
Update MLMD ts lib to 0.22.1

### PR DESCRIPTION
**Pull request by kwasi**
/cc @dushyanthsc @Bobgy 

* Add generated TS lib based on latest protos from #50 
* Update readme to add clarify where Protobuf compilation tools, `protoc` and `protoc-gen-grpc-web` can be found

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,25 +34,26 @@ The package also supports 2 additional build modes:
...
 included in the `src/` directory to allow the build process to succeed without a dependency on
-the Protocol Buffer compiler, `protoc`, being in the system PATH. 
+the Protocol Buffer compiler, `protoc`, being in the system PATH.
 
-If a file in `proto/` is modified or you need to manually re-generate the protos, you'll need to
-add `protoc` to your system's PATH and generate new compiled protos by running:
+If a file in `proto/` is modified or you need to manually re-generate the protos, you'll need to:
+
+* Add `protoc` ([download](https://github.com/protocolbuffers/protobuf/releases)) to your system PATH
+* Add `protoc-gen-grpc-web` ([download](https://github.com/grpc/grpc-web/releases)) to your system PATH
```
Comment (Bobgy) [inline README.md]: Can you mention which version you used last time? So we can avoid version diff causing hard to troubleshoot problems

**Review by Bobgy — COMMENTED** [review]
Thanks @kwasi! This is super helpful

File: README.md (hunk 1 above)
Comment (kwasi) [reply README.md]: I added a comment at the bottom of the section with the protoc and protoc-gen-grpc-web versions.

**Comment by Bobgy** [comment]
Thanks @kwasi!
/lgtm
/approve

**Pull request merged**